### PR TITLE
config: Add a copy of pkg.m4

### PR DESCRIPTION
**Pull request by raffenet**
## What
Add copy of pkg.m4 to config directory.

## Why ?
Users may experience build issues if the installation of pkg-config is
not in the same location as other autotools. Keeping a copy of pkg.m4
within the project is the common workaround.

### CODE REVIEW

**Comment by shamisp** [comment]
@raffenet Conceptually this is good problem to resolve. My only issue is that this is a GPL code.

**Comment by shamisp** [comment]
Found this https://github.com/signal11/hidapi/issues/412  but I would prefer to find something that is inline with BSD.

**Comment by shamisp** [comment]
```
As a special exception to the GNU General Public License, if you
distribute this file as part of a program that contains a
configuration script generated by Autoconf, you may include it under
the same distribution terms that you use for the rest of that
program.
```
Hmm...not all tarballs include configuration script generated by autoconf.

File: config/m4/pkg.m4 [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+dnl MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+dnl General Public License for more details.
+dnl
+dnl You should have received a copy of the GNU General Public License
+dnl along with this program; if not, write to the Free Software
+dnl Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA
+dnl 02111-1307, USA.
+dnl
+dnl As a special exception to the GNU General Public License, if you
+dnl distribute this file as part of a program that contains a
```
Comment (yosefe) [inline config/m4/pkg.m4]: are we good with this license?
@shamisp ?

**Comment by yosefe** [comment]
ok to test

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11464/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 12 of 33 workers (click for details)</summary>

**Note:** the logs will be deleted after 30-Jul-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/907c35c97909f20fdce9dbf17e27ddbcacb7a29d/_main) | :x: FAILURE
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/4cc1a18d94ec1a4a843aedd15ef59c55c3749cb5/hpc-test-althca_W0) | :x: FAILURE
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/a30356d1f71d875aaf8dccd1fce32af0aee1faa7/hpc-test-althca_W1) | :x: FAILURE
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/976a3dd43477ae47745b813f1a4f49fbb012240e/hpc-test-althca_W2) | :x: FAILURE
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/bb21910050e9e283eb5df785811c78c7a7704f98/hpc-test-althca_W3) | :x: FAILURE
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/768113cf68cbbf605ea43a6d30c046d675ccd566/hpc-test-node-gpu_W0) | :x: FAILURE
[hpc-test-node-upstream_W1](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/79d74dd266ec3f5559a7eafc20d45196c79f4b2f/hpc-test-node-upstream_W1) | :x: FAILURE
[hpc-test-node-upstream_W3](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/d74d47913a5b916db3d0a01387929425c6acf11a/hpc-test-node-upstream_W3) | :x: FAILURE
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/a6a982bfeb69cc594fb9eeb4fd73ac425c659ebd/r-vmb-ppc-jenkins_W0) | :x: FAILURE
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/f709caf98e4ed855c84805302d8eeacef42412ca/r-vmb-ppc-jenkins_W1) | :x: FAILURE
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/51a931c8404dc0b3ad92c144337a50fe46c72621/r-vmb-ppc-jenkins_W2) | :x: FAILURE
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/a1bd772d5a7658164d048f097c65210a19dfe150/r-vmb-ppc-jenkins_W3) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/782216a16444b68ac0347a990255a81c7699aa99/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/07695512a81e46bc67f315efe2c4b846c0be8d1e/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/f50f6df8dbf4bb58de01f938c3209aa147cf4c29/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/efdae95243f13b2ade246e7ba98709a3b3ab58e0/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/98352b452a3b75535109f3fae998fb321190e6b6/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/7897bf100f5b863b393dbf1af77df64013d6cb42/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/20875a7e80079fada71dcd92b8699f1cea86ffee/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/3acd0ae805d8001cedcec122e35bfdad77861d75/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/88b9330f7ecbb4bd46f2f5cf5ab67fbe9d31e8a7/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/f8ac607e86cf4ad9eca514eeb43d2a70d9c5936c/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/6861ef2d1b3f49bcc3273dfee2d99e28787306f8/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/012c353e67c652acad4c0ee459197a40bf608053/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/252c992a92f6b1b11c38e50a73ef425413fa28ad/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/38021dd7da18d88b5dd34d8111a2a03d98c4ec1e/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/cba608c76deaf5f30fb473cf4db9ab782ffce546/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/21327fc528cdcafa779fd2b129a73154484982ba/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/7f4d499cd1c155404aafdf6d7c9c0c180f5040b9/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/6299f2bf9af40f4bef82f54e10b8b757532a5def/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/746076a6c20a45cf1024ee19a67dc12c4cb37794/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W0](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/8579c27e72d0da9415c86f2263e7725fc8ed8fc6/hpc-test-node-upstream_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W2](https://gist.githubusercontent.com/mellanox-github/4cfdd82d068baddd8836cf387966368c/raw/57ff21a1d61506960efbba05ffb9653dcf0022a6/hpc-test-node-upstream_W2) | :heavy_check_mark: SUCCESS

File: config/m4/pkg.m4 (hunk 1 above)
Comment (shamisp) [reply config/m4/pkg.m4]: We distribute tarball with autogen.sh script that generates the configuration script by invoking Autoconf. It is subtle difference. Can you check with your legal ? I will do some checks on my side.

Comment (raffenet) [reply config/m4/pkg.m4]: I believe @hppritcha did this for Open MPI. That was my inspiration to submit this PR.

**Comment by yosefe** [comment]
ref not a tree
bot:mlx:retest

File: config/m4/pkg.m4 (hunk 1 above)
Comment (shamisp) [reply config/m4/pkg.m4]: This is true https://github.com/open-mpi/ompi/blob/master/config/pkg.m4

I'm trying to check with experts in this domain.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11484/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 2 of 33 workers (click for details)</summary>

**Note:** the logs will be deleted after 31-Jul-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/41aa2cfbe6c06e72ea5adcc0843d29b082f743b7/_main) | :x: FAILURE
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/7a65a8329482b503d143e0200fc004296ca2c419/hpc-test-node-gpu_W2) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/c89b16c7b8cea132d903bece409d71b4b4b5599f/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/750db2574d3d4a3f6c69364a8678a0f6815a053f/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/b2aa00c31062df97016bb4407530b0218e4cbfd9/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/45a14df2ddc4bbba80bee46d5bdf8d512b17ce9b/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/94ef6a8402752960292015c0f3ec55d5b52344c5/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/adff70086f0a051bf97904b950dba8a21248dbe3/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/853e6c72b96445d4f0f2f29168905f312d690e84/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/4dd1dc36408d1aaa338d8b5107189ea8113ed12f/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/fbab17c7b70353c8fba47d0075b2a71d080d379d/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/d83a4d2d55471b549bea1f24e9e51de16f50368a/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/748649b608f8a213570eb1eaf71a830f555fa874/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/38cb434aefb47e952693919a9f87660fbb8ebcb3/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/80e2f5a9cd3dcd6b0ff460ec6e9a87773c181416/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/d12fbb32f09cc09829a16cea1c447de8531be463/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/311cbef962047a0c3f93e24a129cb1f2560f3a9e/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/a679eb9ab1b8235b98247e6c6e695865247bb38c/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/1775a5ffdd6718440c9cca7e183abb758b750e0c/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/4c15295a2d00459dad637ed46badb2629e33edc7/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/899ecb4358ddcbe750ad9c5090ee702dd727b9b8/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/d1253ad60fbe94c44ec29ede01ab84df2237099a/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/99547b171b645d8d52bd6182e01ae2609f0ddec8/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/cf501ce2b2eeabffb822aec882720eb79d30001b/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/0a743a1c1efc2230a906017092802e53c1979bcc/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W0](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/5a8879d31352724956fec81739689754e012bcc1/hpc-test-node-upstream_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W1](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/4e5302108804239ef45016e7a30b2cd2b83c86d1/hpc-test-node-upstream_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W2](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/069cab101f16fa2dd3cbbab959e9dd794947de4d/hpc-test-node-upstream_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W3](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/d70ca97e99347de4f9bc8d9fc5d2b4ab64c8bacb/hpc-test-node-upstream_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/42b812d5f380349a3607c14be8357f6b46a168cb/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/796515ee04358b4a9d4fad519ecf85c2630d9788/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/5722310a174a44fdcc8bf3dc47d3fde5d0fdf675/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/d441d4c1e4d29773d79c509a62a1ec5b/raw/5148c2d019eccf5682715ac701cd93815ea3f751/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by yosefe** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11509/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 33 workers (click for details)</summary>

**Note:** the logs will be deleted after 01-Aug-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/73e6499c05dff6c1c080a37bad16493da60b40a7/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/716b39e3ec6d739b1da617a337436f2e8761321a/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/b291d1114cf1daaa1c1d8947dd6479c408436dd1/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/66478ec54a38d49a5648a6ac2227e934d1ada25c/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/9088607f625dc434c023afca6a9f59fb23a619a6/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/ed2f74febe851b3f6e2869a4d3073bbf3b88d8a7/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/ba92b028e5d85ac755618195293c29de6294585a/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/9dc0d406dad3c5e8819d82a9dbe6ea1c7e5eeee2/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/0bc512c029202bb0f331b33803ed007bf8e011d6/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/3d5ab36a7e7bf065c7e19372e92e7abe348e9317/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/528c1d92c2793c28ad6197f561fb95927e7cf79d/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/2864ff1af8b38c1b8dfe0ce845bc8fa8271988a1/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/ace610e44b5be242a7ba47a08266f05a32d63952/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/5af15ccfc05e44df8afbff6e9606549e92a7e59d/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/2015a44e8f2af1895750f24dd0102f0107080bb7/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/3fd51f6702573538276c9d56744e886b9387c0bf/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/9864fdc1e2cdf83e20d71e10e7e3f04e294c69be/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/501a4ecfb4704d01a26d9bd120cad80ed5b73a90/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/8b7bea183887e0f57d35e1d6b3741f436dd574f3/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/dc5c2b4211c5ea5668e05000b48927431bdcac9c/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/f0f659fb7d7e42062b5ebc6ef0ebbfb6d82e1f45/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/ce8128bb3adc63e02896edef5bdf90d95eabaa11/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/6440b10ce3de39b4eb3abe02a0cf881314d3f0f5/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/4937ddc8c1131e4c9fcdae77ae03bf7266897226/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/5fccf238ae1f6dcb1feec8cb389d9f3ffeb7fbfb/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W0](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/b7798b1c62cae773063238986fcc2045344faabf/hpc-test-node-upstream_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W1](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/f1b5fb7790436054f246bd3a7a7a6a3ee97e2d14/hpc-test-node-upstream_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W2](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/8d459187868a0d177c6f7af7095d733c3172fb3f/hpc-test-node-upstream_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W3](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/f5167d2b4b1138bc0dcd7a3cb17c89c568778c95/hpc-test-node-upstream_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/9224ecd2d09e41d8bd6863de858d5c0f33e08497/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/f82f47d7e8c413e877657a01ccfac523172369dd/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/0ce175642b6f29d3f26acd9c0d337936ce1aa6a5/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/1518fb9a4d1bbfcd9595500f08ca5df8/raw/39e1634a19e0a3f9bd1979fd10b3f35b603ce939/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by shamisp** [comment]
@yosefe Can you check with your legal folks ?

**Comment by yosefe** [comment]
@shamisp it seems problematic to me, because we don't just distribute this file "as part of a program", but also as standalone file in config/m4
Seems we use pkg config only with UGNI. How about we call it directly as described in https://stackoverflow.com/questions/10220946/pkg-check-modules-considered-harmful and avoid PKG_CHECK_MODULES entirely?

**Comment by shamisp** [comment]
@yosefe what do you mean by "standalone file in config/m4" ? We always distribute it as a part of tarball. 

@hppritcha what was legal advice when you integrated this into OMPI ?

**Comment by yosefe** [comment]
it's part of the tarball, but it's not 'part of a program that contains a configuration script generated by Autoconf'
also, users can clone from github and download pre-autoconf source snaphot

**Comment by shamisp** [comment]
I checked with multiple experts and the advice is NOT to include the file. The exception does not apply. We should remove decency on PKG_CHECK_MODULES macro all together.

**Comment by shamisp** [comment]
Closing this one. @yosefe posted workaround PR here https://github.com/openucx/ucx/pull/3992

**Pull request closed without merging**